src/DistributedComputations/distributed_grids.jl

```diff
@@ -51,10 +54,10 @@ function RectilinearGrid(arch::Distributed,
                          extent = nothing,
                          topology = (Periodic, Periodic, Bounded))
 
-    TX, TY, TZ, global_size, halo, x, y, z =
+    TX, TY, TZ, gsize, halo, x, y, z =
```

Comment:
`global_size` is more verbose, why change?

Reply:
don't want to conflate it with the function `global_size`

Reply:
gotcha!

Reply:
use `global_sz` consistent with how we use `sz` elsewhere